x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx

```diff
@@ -57,10 +57,10 @@ export class TOCEntryActionsPopover extends Component<Props, State> {
   }
 
   componentDidUpdate() {
-    this._checkLayerEditable();
+    this._loadFeatureEditing();
   }
 
-  async _checkLayerEditable() {
+  async _loadFeatureEditing() {
     if (!(this.props.layer instanceof VectorLayer)) {
       return;
     }
```

Comment:
GH won't let me comment lower since it wasn't modified code, but on lines 67, 68 we're making calls to `supportsFeatureEditing` and `this._getCanEditFeatures`. Back-to-back these function names sound like the same idea. Maybe we should change `_getCanEditFeatures` -> `_enableEditFeatures`?

Reply:
replaced "canEditFeatures" with "isFeatureEditingEnabled"